### PR TITLE
Add 'Metrics.update' API wrapper.

### PR DESCRIPTION
**Pull request by tseaver**
~~Uses #1586 as a base.~~

### CODE REVIEW

**Comment by tseaver** [comment]
@dhermes Rebased after merging #1586 to `logging-api` branch.  PTAL

File: docs/logging-usage.rst [hunk 1]
```diff
@@ -186,8 +186,6 @@ Update a metric:
    >>> from gcloud import logging
    >>> client = logging.Client()
    >>> metric = client.metric("robots")
-   >>> metric.exists()  # API call
-   True
```
Comment (dhermes) [inline docs/logging-usage.rst]: Why is this gone?

**Comment by dhermes** [comment]
LGTM, though I think the `.rst` file was fine before?

File: docs/logging-usage.rst (hunk 1 above)
Comment (tseaver) [inline docs/logging-usage.rst]: It seemed irrelevant:  I can put it back.

Comment (tseaver) [inline docs/logging-usage.rst]: 23feb30

**Pull request merged**